src/qt/test/trafficgraphdatatests.cpp

```diff
@@ -0,0 +1,172 @@
+#include "trafficgraphdatatests.h"
+#include "../trafficgraphdata.h"
+#include <algorithm>
+#include <sstream>
+#include <QTime>
+
+void TrafficGraphDataTests::simpleCurrentSampleQueueTests()
+{
+    TrafficGraphData trafficGraphData(TrafficGraphData::Range_5m);
+    for(int i=0; i<TrafficGraphData::DESIRED_DATA_SAMPLES; i++)
```

Comment:
code formatting: spaces around "=" and "<" (this applies to many more cases below)